kernel/environ.mli

```diff
@@ -51,8 +51,9 @@ type globals
 
 type stratification = {
   env_universes : UGraph.t;
-  env_engagement : engagement;
   env_sprop_allowed : bool;
+  env_universes_lbound : Univ.Level.t;
```

Comment:
Should it be here or would it make more sense in the UGraph.t?

Reply:
Not sure, we set it dynamically to elaborate template inductives with `Prop <= i` by default for new universes. The graph itself is pretty agnostic to it (we just insert different constraints in it).